AsociacionDrupalES/landing
# 193 & 163 : All sponsors changes and add new sponsorship levels to the sponsors page.

This pull request includes the following changes:
- Display all sponsors (silver,bronze,gold and platinum) in all pages but with rotation.
- Add Bull , coffee, social & lunch sponsors to the sponsors page.


How to test locally:
- Pull the changes.
- Run composer install from the project root.
- cd web & drush config install
- Create some sponsors of all levels. I suggest to use devel_generate.
- Check the sponsors block at  the front page and other pages.
- Check the page /sponsors with all the sponsors.
